src/settings.py

```diff
@@ -23,6 +23,7 @@ class Settings(BaseSettings):
     USE_GIELLADETECT: bool = False
     GIELLADETECT_LANGS: str = ""
     CUSTOM_PATH: str = ""
+    DIFF_FILES_FOLDER: str = ""
```

Comment:
This env variable is only useful for the diff scripts, whereas `src/` directory contains the code for the REST / FastAPI service. I don't think we should add it here, but we can comment it in `.env.example`.

Reply:
Updated, had to allow extra variables in the Config to prevent error thrown in newer version of pydantic.

Reply:
As we just discussed: it is problematic that the need for this new "allow extra" config in settings appears only when running the actual program, and not during the lint_and_test pipeline. Writing a test that checks that the FastAPI service is configured properly should be straightforward (hopefully!).

Reply:
Added a simple test using the .env.example file which would fail if extra='allow' was not present in the settings configuration.